src/core_plugins/kibana/public/visualize/editor/advanced_toggle.html

```diff
@@ -1,6 +1,6 @@
-<div class="vis-editor-agg-editor-advanced-toggle">
+<div class="visEditorAgg__editor-advanced-toggle">
```

Comment:
Technically this should be `visEditorAgg__editorAdvancedToggle`, but it really doesn't matter per my comment re mass name-changes.